Completion/Linux/Command/_nmcli

```diff
@@ -1,244 +1,369 @@
-#compdef nmcli
+#compdef _nmcli nmcli
 
-local context state line expl
-typeset -A opt_args
+# compatible with nmcli 1.8.4
 
-if [[ -z $_nmcli_version ]]; then
-    _nmcli_version="${"$(_call_program nmcli $words[1] --version)"##*version }"
-fi
+_nmcli() {
+    local line    _arguments -C \
```

Comment:
Have two lines become wrapped here? When using _arguments' -C option, you should also declare:
  local curcontext="$curcontext" state line

Reply:
Yes, that must have been an accidental change prior to the commit.

With regards to `local curcontext="$curcontext" state line`, should I still declare `curcontext` and `state` if I don't use them in this function?

Reply:
Yes. When you're using _arguments' state (:->args) syntax, you are asking _arguments to modify those variables. The value of $curcontext is being used indirectly in later calls _description when looking up styles. Note that curcontext needs a coy of the existing value.

Reply:
Thanks for the explanation!